packages/app/src/specs/InlineRunAllSpecs.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <Tooltip
+    placement="right"
+    class="h-full truncate"
+    data-cy="tooltip"
+  >
+    <button>
+      <IconActionPlaySmall
+        size="16"
+        stroke-color="gray-700"
+        fill-color="transparent"
+        hocus-stroke-color="indigo-500"
+        hocus-fill-color="indigo-100"
+        class="inline-flex align-text-bottom"
+        data-cy="play-button"
+        @click.stop="emits('runAllSpecs')"
```

Comment:
I think this click handler should be on the button, not the icon. When I use the keyboard on the button it doesn't rerun specs for me, it just toggles the folder open and closed.

Reply:
Hmm, moving the handler didn't help, I think because the parent is using `@click.capture` to manage the toggle. That code would need some extra logic to not toggle if the click event came from the button.

Reply:
I moved the listener anyway, it should be on the button. There is a follow up for better a11y.